website/project/model.py

```diff
@@ -4588,3 +4622,36 @@ def approve(self, user):
     def reject(self, user):
         self.approval.reject(user)
         self.approval.save()
+
+
+@unique_on(['destination_node', '_id'])
+class MailingListEventLog(StoredObject):
```

Comment:
How does this relate to the rest of the logging system? Does it? I would expect to see some subclassing or something ...